packages/background/src/Background/index.ts

```diff
@@ -0,0 +1,198 @@
+import { type PropType, computed, defineComponent, h } from 'vue-demi'
+
+import { useVueFlow } from '@vue-flow/core'
+
+import { type BackgroundProps, BackgroundVariant } from '../types'
+import { DefaultBgColors, DotPattern, LinePattern } from '../Pattern'
+
+import { transformForCompatibleAttrs, transformForCompatibleProps } from '../utils'
+
+const Background = /* #__PURE__ */ defineComponent({
+  name: 'Background',
+  compatConfig: { MODE: 2 },
+  props: {
```

Comment:
Alas, I thought this might be the result of the syntax change, which imo, increases maintenance a lot.

Instead of `defineProps<Props>()` we now end up with this mess :(

I assume due to Vue 2 compatibility using the `defineComponent<Props, Emits>({ ... })` will not work here, will it?